From 315bb66486118ec189526d8eb1c5e98031d78bbf Mon Sep 17 00:00:00 2001
From: Yang Fan F <fan.f.yang@intel.com>
Date: Wed, 29 Apr 2015 01:57:01 +0800
Subject: [PATCH 06/28] Revert "atomisp2: Fix pm_qos handling in probe()'s
 failure path"

This reverts commit 95e72fec24c11fa0401d8ea604a0f9d1e75e9fab.

Change-Id: I8346bda7b3fdaa5982688f3961bd8b47efd4def3
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-9279
Reviewed-on: https://android.intel.com:443/440492
---
 .../camera/drivers/media/pci/atomisp2/atomisp_driver/atomisp_v4l2.c    | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/drivers/external_drivers/camera/drivers/media/pci/atomisp2/atomisp_driver/atomisp_v4l2.c b/drivers/external_drivers/camera/drivers/media/pci/atomisp2/atomisp_driver/atomisp_v4l2.c
index 2988b96..732cf2a 100644
--- a/drivers/external_drivers/camera/drivers/media/pci/atomisp2/atomisp_driver/atomisp_v4l2.c
+++ b/drivers/external_drivers/camera/drivers/media/pci/atomisp2/atomisp_driver/atomisp_v4l2.c
@@ -1582,6 +1582,9 @@ enable_msi_fail:
 fw_validation_fail:
 	release_firmware(isp->firmware);
 load_fw_fail:
+#ifdef CONFIG_GMIN_INTEL_MID
+	pm_qos_remove_request(&isp->pm_qos);
+#endif
 	pci_dev_put(isp->pci_root);
 	return err;
 }
-- 
1.9.1

